src/Bossbat.js

```diff
@@ -127,10 +127,9 @@ export default class Bossbat {
         throw new Error(`Unknown interval of type "${typeOfEvery}" passed to hire.`);
       }
     } else if (definition.cron) {
-      const nextTime = parseExpression(definition.cron, { iterator: false }).next().getTime();
-      const cronTimeout = nextTime - Date.now();
-      // Force timeout to be at least 1:
-      timeout = cronTimeout >= 1 ? cronTimeout : 1;
+      const iterator = parseExpression(definition.cron, { iterator: false });
+      const cronTimeout = iterator.next().getTime() - Date.now();
+      timeout = cronTimeout > 0 ? cronTimeout : iterator.next().getTime() - Date.now();
```

Comment:
Instead of replicating the same iterator code twice, could we just wrap this into a function we can call?